### PR TITLE
refactor: consistent naming of injection tokens

### PR DESCRIPTION
**Pull request by markostanimirovic**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Name/description of injection tokens is inconsistent.

**@ngrx/component-store**
```typescript
export const initialStateToken = new InjectionToken('ComponentStore InitState');
```

**@ngrx/data**
```typescript
export const ENTITY_EFFECTS_SCHEDULER = new InjectionToken<SchedulerLike>(
  'EntityEffects Scheduler'
);

export const ENTITY_METADATA_TOKEN = new InjectionToken<EntityMetadataMap>(
  '@ngrx/data/entity-metadata'
);
```

**@ngrx/effects**
```typescript
export const EFFECTS_ERROR_HANDLER = new InjectionToken<EffectsErrorHandler>(
  'ngrx/effects: Effects Error Handler'
);
```

etc.

## What is the new behavior?

Name/description of injection tokens is consistent.

Format: `@ngrx/module-name Title Cased Description`

## Does this PR introduce a breaking change?

```
[x] Yes
[ ] No
```

### CODE REVIEW

**Review by alex-okrushko — APPROVED** [review]
Thanks 👍

**Comment by alex-okrushko** [comment]
@brandonroberts 
I see the failure of
```sh
> ng run data:test 
Nx could not find process's output. Run the command without --parallel.
```

I restarted the workflow, hope it will fix it.

File: modules/component-store/src/component-store.ts [hunk 1]
```diff
@@ -32,7 +32,9 @@ export interface SelectConfig {
   debounce?: boolean;
 }
 
-export const initialStateToken = new InjectionToken('ComponentStore InitState');
+export const INITIAL_STATE_TOKEN = new InjectionToken(
```
Comment (timdeschryver) [inline modules/component-store/src/component-store.ts:35]: Forgot about this PR.
I was thinking if this could be seen as a breaking change 🤔

**Review by timdeschryver — APPROVED** [review]

File: modules/component-store/src/component-store.ts (hunk 1 above)
Comment (brandonroberts) [reply modules/component-store/src/component-store.ts:35]: That token is part of the public API, so I would say yes

Comment (timdeschryver) [reply modules/component-store/src/component-store.ts:35]: I'll tag it for v11 :)

Comment (alex-okrushko) [reply modules/component-store/src/component-store.ts:35]: ComponentStore's token was not supposed to be exposed 😐 It's necessary for ViewEngine backwards-compatibility.
I'm fine with leaving it for v11, at the same time I'm suspecting that our next release would be v11 🙂

**Pull request merged**